tools/docs/public_site.py

```
@@ -0,0 +1,466 @@
+        raise ValueError("Platform support page must be revisited because iosX64 is now published")
+
+    lines = [
+        "# Platform support",
+        "",
+        "These values are generated from the current build and platform bridge sources so the public matrix cannot silently drift.",
+        "",
+        "| Surface | Current support | Evidence boundary |",
+        "| --- | --- | --- |",
+        f"| Android client libraries | `minSdk {values['androidMinSdk']}`, compiled with SDK {values['androidCompileSdk']} | Compilation and host tests do not prove a provider-backed ceremony |",
+        f"| Compose PRF sample | `minSdk {values['sampleMinSdk']}` | The sample minimum is not the base library minimum |",
+        f"| iOS targets | `iosArm64`, `iosSimulatorArm64`; no `iosX64` | Simulator compilation does not prove device entitlements or iCloud Keychain behavior |",
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)